documentation/docs/release-notes/2.43.2-1.md

```diff
@@ -0,0 +1,154 @@
+# Percona Monitoring and Management 2.43.2-1
+
+**Release date**: March 12th, 2025                                                                  
+Percona Monitoring and Management (PMM) is an open-source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB. 
+
+It provides tools to observe database health, analyze performance trends, troubleshoot issues, and execute database management tasks—whether your databases are on-premises or in the cloud.
+
+## SEVERITY: CRITICAL - IMMEDIATE ACTION REQUIRED
+
+A critical vulnerability has been identified in PMM Open Virtual Appliance (OVA) installations that enables unauthorized `root` access and potential exposure of system credentials. This vulnerability is tracked as [CVE-2025-26701](https://cve.mitre.org/cgi-bin/cvename.cgi?name=2025-26701).
+
+### Immediate actions required
+
+1. UPGRADE IMMEDIATELY to PMM 2.43.2-1 or to the latest PMM 2 release [PMM 2.44.0-1](../release-notes/2.44.0.1.md).
+2. CHANGE ALL CREDENTIALS for connected services.
+3. AUDIT ACCESS LOGS for potential unauthorized access.
+
+### Vulnerability details
+This vulnerability stems from default service account credentials in OVA provisioning that enables:
+
+ - unauthorized SSH access
+ - privilege escalation to `root` via `sudo` capabilities
+ - potential exposure of service credentials and configurations
+
+### Affected installations  
+The table below lists currently known affected deployments. We will update it if additional deployments are identified as vulnerable:
+
+| Affected deployments                     | Version | Notes |
+|-------------------------------------------|---------|-------|
+| PMM Open Virtual Appliance (OVA) installations | ≥ 2.38  |       |
```

Comment:
I assume it would be better to have a range here like <2.44.0 pr something like this as from ≥ 2.38 I assume 2.44 is also affected